src/Factory/FormErrorFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CoopTilleuls\SyliusQuickImportPlugin\Factory;
+
+use CoopTilleuls\SyliusQuickImportPlugin\Exception\InvalidFileExtensionException;
+use CoopTilleuls\SyliusQuickImportPlugin\Exception\ImporterException;
+use CoopTilleuls\SyliusQuickImportPlugin\Exception\InvalidFileFormatException;
+use CoopTilleuls\SyliusQuickImportPlugin\Exception\MissingDataException;
+use CoopTilleuls\SyliusQuickImportPlugin\Exception\MissingFileException;
+use Symfony\Component\Form\FormError;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class FormErrorFactory implements FormErrorFactoryInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
```

Comment:
PHP 7.4?

Reply:
IMHO it's better to not use the last PHP version. It will be more easy to install the pugin like that